virtool/jobs/data.py

```diff
@@ -92,11 +92,8 @@ async def _find_beta(self, query: Mapping) -> Document:
         }
         """
         state = query.get("state")
-        archive = query.get("archived")
         term = query.get("find")
-
-        if archive is not None:
-            archive = bool(archive == "True")
+        archive = virtool.utils.to_bool(query["archived"]) if "archived" in query else None
```

Comment:
Remove this from PR. I am working on a fix for this and will review it with you.